tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -668,7 +668,7 @@ Engine built-in types
 
 :ref:`Color <class_Color>`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
-
+Registering named classes
```

Comment:
This looks like a mistake. The header already exists.

https://github.com/godotengine/godot-docs/blob/57ef6737573158a2033b34ef9fe4bdeb14e07f80/tutorials/scripting/gdscript/gdscript_basics.rst?plain=1#L1574-L1580